sdk/communication/Azure.Communication.MediaComposition/src/common.json

```diff
@@ -0,0 +1,172 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Communication",
+        "description": "Communication Services API common types.",
+        "version": "2022-07-13"
+    },
+    "paths": {},
+    "definitions": {
+        "CommunicationErrorResponse": {
```

Comment:
CommunicationErrorResponse and CommunicationIdentifierModel should use a relative path to reference the common.json from the media composition swagger.

Reply:
Yes, will do that as soon as the I link to the swagger in the specs repo